static/app/components/profiling/arrayLinks.tsx

```diff
@@ -25,7 +25,7 @@ function ArrayLinks({items}: ArrayLinksProps) {
       {items.length > 1 &&
```

Comment:
small note - ArrayLinks seems like a very generic name, can we rename this to something more specific?